src/common/components/repository-row/index.js

```diff
@@ -407,6 +410,7 @@ export class RepositoryRowView extends Component {
       return (
         <form onSubmit={this.onRegisterSubmit.bind(this, snap.gitRepoUrl)}>
           <input
+            autoFocus={true}
```

Comment:
Added `autofocus` to register name inputs, so they are focused when dropdown is opened and it is more clear that user can edit them and not just blindly register suggested name.